### PR TITLE
Remove btn-default when use delete button

### PR DESCRIPTION
**Pull request by guandai**
By default,  delete button should not have `btn-default` class

### CODE REVIEW

File: lib/bootstrap_builders/class_attribute_handler.rb [hunk 1]
```diff
@@ -9,6 +9,14 @@ def add(class_argument)
     @classes += convert_to_array(class_argument)
   end
 
+  def remove(class_argument)
+    if class_argument.is_a?(Array)
+      @classes = @classes - class_argument
```
Comment (kaspernj) [inline lib/bootstrap_builders/class_attribute_handler.rb]: I believe you can write this just a tiny bit shorter. Do you mind giving it a try?

```ruby
@classes -= class_argument
```

**Comment by kaspernj** [comment]
Other than my comment, you changes look superb :-)

File: lib/bootstrap_builders/class_attribute_handler.rb (hunk 1 above)
Comment (guandai) [inline lib/bootstrap_builders/class_attribute_handler.rb]: Updated in 44e60fe

**Comment by kaspernj** [comment]
@guandai Do you mind adding an `expect` to specify your change to make sure someone doesn't accidentally revert it later here? :-)

https://github.com/kaspernj/bootstrap_builders/blame/master/spec/features/bootstrap_builders/button_spec.rb#L18

**Comment by guandai** [comment]
@kaspernj  Added in 6006238

**Pull request merged**